packages/flutter/lib/src/widgets/raw_menu_anchor.dart

```diff
@@ -405,17 +571,48 @@ mixin _RawMenuAnchorBaseMixin<T extends StatefulWidget> on State<T> {
   @protected
   void open({Offset? position});
 
-  /// Close the menu.
+  /// Close the menu and all of its children.
+  ///
+  /// If `inDispose` is true, the menu will close without rebuilding its parent.
+  ///
+  /// Unless the menu needs to be closed immediately, [handleCloseRequest] should be
+  /// called instead of [close]. Doing so allows subclasses to control how the
+  /// menu is closed.
```

Comment:
This paragraph always bugs me. I find it confusing because this method can be called by both the base class and subclasses, but this paragraph is only useful to the base class. What do you think of:
```suggestion
  /// The `close` method might be called by the base class `_RawMenuAnchorBaseMixin` and its subclasses.
  /// `_RawMenuAnchorBaseMixin` should only call `close` if the menu needs to be closed immediately;
  /// otherwise, call `handleCloseRequest` to allow parent widgets to control how the
  /// menu is closed.
```

Reply:
Subclasses also implies a subclass of _RawMenuAnchorState or _RawMenuAnchorGroupState, but we can just take this paragraph out altogether since "open" and "close" aren't actually used by the base class. Developers just need to know is that open() shows the overlay and close() hides the overlay.

Reply:
Well that’s the point: There are no subclasses of _RawMenuAnchorState or _RawMenuAnchorGroupState. The entire class tree is private and will only reach as far as this files contains.

Reply:
Hmm.. so is it okay to remove the paragraph altogether?

Reply:
Yeah, I think that works too. Again, this is private doc and doesn't require too detailed guidance.

Reply:
Done!